language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/WaitActionNodeContext.java

```diff
@@ -137,7 +141,14 @@ public void sort(BallerinaCompletionContext context, WaitActionNode node, List<L
                     rank = 1;
                 } else if (symbol.kind() == VARIABLE
                         && ((VariableSymbol) symbol).typeDescriptor().typeKind() == TypeDescKind.FUTURE) {
-                    rank = 2;
+                    Optional<TypeSymbol> typeSymbol 
```

Comment:
Shall we use the genSortTextByAssignability() method in the SortingUtil class

Reply:
We cannot use it because `isCompletionItemAssignable()` takes in a `LSCompletionItem` and from that it takes the `SymbolCompletionItem`. But here we get the `typeSymbol` from the `FutureTypeSymbol`.